src/main/java/cms/gov/madie/measure/services/TestCaseService.java

```diff
@@ -412,17 +395,28 @@ private TestCaseImportOutcome validateTestCaseJsonAndCreateTestCase(
           QiCoreJsonUtil.getPatientName(testCaseImportRequest.getJson(), "given");
       log.info(
           "Test Case title + Test Case Group:  {}", patientGivenName + " " + patientFamilyName);
-
       TestCase newTestCase =
           TestCase.builder().title(patientGivenName).series(patientFamilyName).build();
       List<TestCaseGroupPopulation> testCaseGroupPopulations =
           QiCoreJsonUtil.getTestCaseGroupPopulationsFromMeasureReport(
               testCaseImportRequest.getJson());
-
-      matchCriteriaGroups(testCaseGroupPopulations, measure.getGroups(), newTestCase);
-
+      boolean matched =
+          matchCriteriaGroups(testCaseGroupPopulations, measure.getGroups(), newTestCase);
+      String warningMessage = null;
+      if (!matched) {
+        warningMessage =
+            "For Test Case "
+                + testCaseImportRequest.getPatientId()
```

Comment:
"The measure populations do not match the populations in the import file. The Test Case has been imported, but no expected values have been set.";

I believe this message is good enough and we can drop Patient Id, as it will be redundant in the UI.

Reply:
@RohitKandimalla Patient Id is removed from the warning message. Thanks!